node/src/components/block_proposer/tests.rs

```diff
@@ -11,31 +13,36 @@ use crate::{
 };
 
 use super::*;
-use casper_types::standard_payment::ARG_AMOUNT;
 
 fn default_gas_payment() -> Gas {
     Gas::from(1u32)
 }
 
+// gas_price used for test deploys
+const TEST_GAS_PRICE: u64 = 1;
```

Comment:
Might be better to use a gas price other than 1 in all the tests, to make it more likely that we'd detect if we forgot to multiply or divide somewhere.